### DISCUSSION TITLE
UI Bug RE: Time of last comment

### DISCUSSION THREAD

**Discussion by Hanse00** [community/community#2855] [Discussions] [answered]
Hey everyone!

Perhaps I'm just misunderstanding the choice of language here, but it seems like the discussions has a minor UI bug.

Each discussion has a line along the lines of: `[username] started [time] ago in [category]` shown below the title.
However, once that discussion receives any comments, it appears like the time here is actually tracking when the last comment was made, *not* when the discussion was actually started.

As an example, in this very category, this discussion was created 37 minutes ago, but commented on 9 minutes ago, yet the language in the UI suggests to me that the discussion was started 9 minutes ago:
![Screen Shot 2020-12-08 at 12 18 54 PM](https://user-images.githubusercontent.com/1231254/101537218-f6c16080-394f-11eb-8bf1-5dd055e5f7c9.png)

![Screen Shot 2020-12-08 at 12 19 01 PM](https://user-images.githubusercontent.com/1231254/101537256-05a81300-3950-11eb-855a-dbc23ae2017b.png)

**Comment by becca** [comment, marked as answer]
Thank you for reporting this. This space is for feedback but we are tracking a fix for this bug and will let you know when a resolution has landed.

**Reply by Hanse00** [reply]
Sorry! It wasn't immediately obvious to me where to file it otherwise 😄

**Reply by becca** [reply]
You're good! Super appreciate all of your contributions @Hanse00 🏆

**Reply by becca** [reply]
This should now be fixed. Thank you @Hanse00 for the report.
